src/runtime_src/core/tools/xbmgmt2/flash/firmware_image.cpp

```diff
@@ -608,7 +608,14 @@ firmwareImage::firmwareImage(const char *file, imageType type) :
         in_file.seekg(0);
         in_file.read(mBuf, bufsize);
     }
-    this->str(mBuf);
+
+// rdbuf doesn't work on windows and str() doesn't work for ospi_versal on linux
+#ifdef __GNUC__
```

Comment:
Would like to know more about this issue.

Reply:
Sure. Will discuss on call